src/figuresClasses.ts

```diff
@@ -1,19 +1,76 @@
 export interface Figure {
-
+  shape: 'triangle' | 'circle' |'rectangle',
+  color: 'red' | 'green' | 'blue',
+  getArea: Function,
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  constructor(public color: 'red' | 'green' | 'blue', public a: number,
+    public b: number, public c: number, public shape: 'triangle' = 'triangle') {
+    const theLongest = Math.max(a, b, c);
+    const sides = [a, b, c];
+    const shorterSides = sides.filter((x) => x !== theLongest);
 
-}
+    if (theLongest >= shorterSides[0] + shorterSides[1]) {
```

Comment:
it looks more complicated and complex than just checking these 3 cases manually

Reply:
maybe you can try to sort it first and then you will know which one is longed, and you will also have shorter sides